networkx/algorithms/isomorphism/vf2pp.py

```diff
@@ -1,41 +1,42 @@
 import collections
 
 import networkx as nx
-from networkx.algorithms.isomorphism.VF2pp_helpers.candidates import find_candidates
-from networkx.algorithms.isomorphism.VF2pp_helpers.feasibility import feasibility
-from networkx.algorithms.isomorphism.VF2pp_helpers.node_ordering import matching_order
-from networkx.algorithms.isomorphism.VF2pp_helpers.state import (
+from networkx.algorithms.isomorphism.vf2pp_helpers.candidates import find_candidates
+from networkx.algorithms.isomorphism.vf2pp_helpers.feasibility import feasibility
+from networkx.algorithms.isomorphism.vf2pp_helpers.node_ordering import matching_order
+from networkx.algorithms.isomorphism.vf2pp_helpers.state import (
     restore_state,
     update_state,
 )
 
 
-def VF2pp(G1, G2, G1_labels, G2_labels):
+def VF2pp(G1, G2, node_labels=None):
```

Comment:
Whlie we are refactoring, we should discuss the choice of G1 and G2. In the VF2 implementation, for the subgraph isomorphism G2 is mapped to a subgraph of G1.  That is confusing because the mapping is documented as mapping from G1 to G2. The tests for mapping show that at least for the isomorphism problem `mapping` does map G1 to G2. I don't see how that changes for the subgraph problems, but I don't see any tests of the direction of the mapping for the subgraph problem. Anyway, we should check.  The documentation seems to suggest that the current VF2 looks for an isomorphism mapping a subgraph of G1 to all of G2.

Personally, I prefer the syntax we are currently using: mapping G1 to G2 and if we want a subgraph problem we look at subgraphs of G2.  But we should figure out what the current API is and decide whether the backward incompatibility is worth the change (if there is actually a change -- which I think there would be).  

**Second topic:  node label attribute names**
Might the user have different label attribute names for the two graphs? 
That is, should we allow `node_labels1=None, node_labels2=None`?

Then what would the default behavior be if only one is specified? We could raise an exception -- but perhaps for convenience, if node_labels1 is specified, then the default for node_label2 is the value of node_label2.
hmmm... or is that too confusing?